expyfun/stimuli/_hrtf.py

```diff
@@ -151,3 +151,67 @@ def convolve_hrtf(data, fs, angle, source='barb'):
         brir = [resample(b, fs, brir_fs) for b in brir]
     out = np.array([np.convolve(data, brir[o]) for o in order])
     return out
+
+    
+def make_sym(x):
+    """
+    forces symmetry around zero
+    """
+    n = x.shape[-1]
+    x[..., :-((n + 1) // 2):-1] = np.conj(x[..., 1:(n + 1) // 2])
+    if np.mod(n, 2) == 0:
+        x[..., n // 2] = 0
+    return x
+    
+
+def interp_hrtf(angle, fs):
+    """
+    This function takes the two nearest known HRTFs (in the form of a BRIR) and
+    uses them to calculate an HRTF (again, as a BRIR) between them for any
+    given azimuthal angle between 0 and 90 degrees.
+
+    Parameters
+    ----------
+    angle : float
+        The desired angle of the interpolated BRIR in degrees. The angle must
+        be between -90 and 90 degrees.
+    fs : int
+        The sampling frequency of the BRIR
+
+    Returns
+    -------
+    brir : 2 x 184 matrix containing interpolated BRIR for the given angle
```

Comment:
please make this conform to the NumPy docstring standard (look at other expyfun functions for examples)

Also, please add a `References` section with the referenced paper